CHANGELOG.md

```diff
@@ -2,6 +2,531 @@
 
 List of the most important changes for each release.
 
+## 0.18.0
+
+### High level overview
+Release 0.18.0 introduces improvements and new features in the **Coach** experience to make it easier to find materials when creating a lesson or quiz.
+
+### Added
+**New Feature: Coaches can now use metadata filtering when creating lessons and quizzes**
+
+Updates to lessons and quizzes help coaches find what they need faster. Coaches can use filters like **activity type** (video, audio, reading materials), **language**, **level**, **category** (school subjects, vocational materials), and more. By combining a variety of filters, coaches can quickly locate specific and relevant materials they need.
+
+Noteworthy technical updates include rearchitecting our Coach plugin pages to support the navigation updates, extending the routing structure in coach to use nested routes and a side-panel style UX in lessons, and upgrading KDS to provide a variety of bug fixes and introduce the new `KCard` component.
```

Comment:
You mention "support the navigation updates" here, but have not mentioned them above. It's a fairly significant user change we should probably mention.